tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -90,6 +91,10 @@ <h3>No profile data was found.</h3>
       <input-pipeline-analyzer run="[[_getSelectedDataset(_datasets, currentSelected)]]">
       </input-pipeline-analyzer>
     </template>
+    <template is="dom-if" if="[[_isCurrentTool(_datasets, currentSelected, 'overview_page')]]" restamp="true">
```

Comment:
Why is `restamp="true"` needed? Do other elements within this page require the content here to be in the DOM even when the content is hidden?

Reply:
Leave it to the default (which is "false") now.